docs/webserver.md

```diff
@@ -0,0 +1,21 @@
+# Webserver
+
+The main web server hosts:
```

Comment:
```suggestion
The main web server (web02.rackspace.theforeman.org) hosts:
```

Reply:
I think we should have a table at the beginning of the page, with all the technical details, and not call out the hostname here.

smth like:
| | web02.rackspace.theforeman.org |
| - | - |
| type | OpenStack VM |
| OS | CentOS 7 |
| CPUs | 2 |
| RAM | 2GB |
| Storage | /dev/xvda (40GB): root, /dev/xvdb (100GB) + /dev/xvdc (50GB): data |

Reply:
I've been wondering about that - we have that info in Foreman so is it really that relevant? I always like it when the reasons are documented rather than the result.

Reply:
While I agree the data is in Foreman, I kinda like some basics being available here so that it's available when Foreman is not.